subgraph-js/src/directives.ts

```diff
@@ -30,6 +31,10 @@ export const KeyDirective = new GraphQLDirective({
     fields: {
       type: new GraphQLNonNull(GraphQLString),
     },
+    resolvable: {
```

Comment:
It's a little weird to me that we have to add to here as well as in the keyDirectiveSpec in `federationSpec.ts`. Is there a way to derive our GraphQLDirectives from our DirectiveSpecifications or vice versa so that we don't have to maintain both sets of types?

Reply:
It is a bit unfortunate and I hope we can eventually fix that, but I'd rather punt on that for now because 1) `subgraph-js` and `internals-js` don't currently depend on each other and it's worth thinking if we have to have a dependency and 2) because both use slightly different abstractions, the we'd probably have to hard-code a bunch of equivalences to derive one from the other, and that's probably a bit of effort to take in (aka, this probably deserve to be a separate dedicated task at least).

Reply:
Definitely didn't mean that we should address it now, I was just asking in general.